src/Microsoft.TemplateEngine.Edge/LocalizableStrings.resx

```diff
@@ -215,6 +215,9 @@ Details: {1}.</value>
     <value>Failed to check the update for the package {0}.
 Details: {1}.</value>
   </data>
+  <data name="Scanner_Error_TemplatePackageLocationIsNotSupported" xml:space="preserve">
+    <value>Template package location {0} is not supported, or doesn't exist</value>
```

Comment:
Should end with '.'
```suggestion
    <value>Template package location {0} is not supported, or doesn't exist.</value>
```

Reply:
This is intentional. This line is later put into `Details: {1}.`  So, I need to remove the dot in one of two places to avoid two dots in the end of the sentence. 
To avoid fixing the another localized string, I removed it from here.

Reply:
Please remove it. According to guideline the exception should end with period. https://docs.microsoft.com/en-us/dotnet/standard/exceptions/best-practices-for-exceptions#include-a-localized-string-message-in-every-exception

Reply:
I see, I shall remove the dot from the other place then.